recipes-example/example/sdl-test.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Example SDL test application"
+
+LICENSE = "CLOSED"
+
+DEPENDS = "virtual/libgles2 virtual/libsdl2"
+
+SRC_DAC_EXAMPLES = "home/maricnh/workspace/dac-examples-src"
+
+S = "${WORKDIR}/${SRC_DAC_EXAMPLES}/sdl-test/"
+
+SRC_URI = " \
```

Comment:
need to be replaced by remote path

Reply:
The https://github.com/stagingrdkm/dac-examples-src pool request has been merged. Could you update the link for the app ?